mainapp/templates/mainapp/food.html

```diff
@@ -0,0 +1,257 @@
+{% extends 'mainapp/base.html' %}
```

Comment:
Use a single HTML page for all the categories instead of different ones since everything is the same except the data passed.